invenio_grobid/api.py

```diff
@@ -36,7 +40,8 @@
 def process_pdf_stream(pdf_file):
     """Process a PDF file stream with Grobid, returning TEI XML results."""
     response = requests.post(
-        url=os.path.join(cfg.get("GROBID_HOST"), "processFulltextDocument"),
+        url=os.path.join(current_app.config.get("GROBID_HOST"),
```

Comment:
Yeah, It's better I think (and avoids us from having `invenio_base`/`invenio.base` imports).